src/main/java/baritone/pathing/movement/movements/MovementDiagonal.java

```diff
@@ -213,6 +213,21 @@ public static void cost(CalculationContext context, int x, int y, int z, int des
         res.z = destZ;
     }
 
+    protected boolean safeToCancel(MovementState currentState) {
```

Comment:
this isn't used anywhere? unless I'm missing something and it's an override

Reply:
It is an override from Movement and called by the default implementation of safeToCancel() in Movement